src/main.rs

```diff
@@ -117,9 +124,20 @@ fn main() -> Result<()> {
     // Read configuration from disk or generate a default one.
     let config = Config::get_or_create()?;
     let _ = fs::create_dir(&config.temporary_project_dir);
-    let tmp_dir = Builder::new()
-        .prefix("tmp-")
-        .tempdir_in(&config.temporary_project_dir)?;
+
+    // Create the temporary directory
+    let tmp_dir = {
+        let mut builder = Builder::new();
+
+        if cli.worktree_branch.is_some() {
+            builder.prefix("wk-");
+        } else {
+            builder.prefix("tmp-");
+        }
```

Comment:
I don't think this is particularly relevant... but it doesn't hurt much the readability :woman_shrugging: